meilisearch/landing
# Add job offers block

# Pull Request

## Related issue
Closes #483

## What does this PR do?
- Addition of a new `color` props to the `Tag` component
- Creation of a new block to display the job offers list + its Storybook story

Note : ⚠️ The `TODO : Add tests` in the code is completely normal. This will be done in an other PR

## How to test

Go to [the deploy branch](https://landing-git-addjoboffersblock-meili.vercel.app/about-us) and check the `Join us!` block
You can also `yarn storybook` and go to [Storybook](http://localhost:6006/?path=/story/blocks-joboffers--default)

## PR checklist
Please check if your PR fulfills the following requirements:

Thank you so much for contributing to Meilisearch!
